### ISSUE TITLE
unable to build using prod

### ISSUE THREAD

**Issue by gemal** [moff/angular2-flash-messages#31]
when I try to use --aot to build a project with FlashMessagesModule I get an error

`ERROR in Error: Unexpected value ‘FlashMessagesModule in x/node_modules/angular2-flash-messages/module/module.d.ts’ imported by the module ‘AppModule in x/src/app/app.module.ts’. Please add a @NgModule annotation.`

does FlashMessagesModule not work with aot yet?

**Comment by Dinistro** [comment]
The problem is, that there are no ngc metadatas, that are requiered for the usage with `--aot` or `--prod`.

**Comment by gemal** [comment]
is that something that the angular2-flash-messages should provide or?

**Comment by peymannj** [comment]
@gemal I think so. Anyway I got a same problem.

**Comment by Dinistro** [comment]
@gemal Yeah, they need to kick the ngc (angular compiler) and provide the generated metadata for the angular-cli.

**Comment by moff** [comment]
@gemal , @peymannj published updated package. Please update and let me know if problem still persists.

**Comment by peymannj** [comment]
@moff  I updated to 2.0.1 and also I removed forRoot()  function. Now I get new error message : 

`ERROR in Error: Error encountered resolving symbol values statically. Could not resolve @angular/core relative to D:/Project/node_modules/angular2-flash-messages/module/module.d.ts., resolving symbol FlashMessagesModule in D:/Project/node_modules/angular2-flash-messages/module/module.d.ts, resolving symbol FlashMessagesModule in D:/Project/node_modules/angular2-flash-messages/module/module.d.ts`

**Comment by moff** [comment]
@gemal , @peymannj Hm, I think this problem has something to do with this https://github.com/angular/angular-cli/issues/4647

**Comment by moff** [comment]
@gemal , @peymannj check this comment https://github.com/angular/angular-cli/issues/4647#issuecomment-305967234

**Comment by peymannj** [comment]
This line fixed the problem  
`"paths": { "@angular/*": ["../node_modules/@angular/*"] }`
